src/main/kotlin/uk/gov/justice/digital/hmpps/electronicmonitoringdatastoreapi/resource/OrderController.kt

```diff
@@ -1,80 +1,72 @@
-package uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.controllers
+package uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.resource
 
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.http.ResponseEntity
-import org.springframework.security.access.AccessDeniedException
 import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.security.core.Authentication
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.RequestHeader
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.AthenaQueryResponse
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.KeyOrderInformation
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.model.OrderInformation
 import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.repository.OrderInformationRepository
+import uk.gov.justice.digital.hmpps.electronicmonitoringdatastoreapi.service.internal.AuditService
 
 @RestController
-// @PreAuthorize("hasRole('ELECTRONIC_MONITORING_DATASTORE_API_SEARCH') and hasAuthority('ROLE_EM_DATASTORE_GENERAL_RO')")
+// @PreAuthorize("hasRole('ROLE_EM_DATASTORE__SEARCH')")
 @PreAuthorize("hasRole('ELECTRONIC_MONITORING_DATASTORE_API_SEARCH')")
 @RequestMapping(value = ["/orders"], produces = ["application/json"])
 class OrderController(
   @Autowired val repository: OrderInformationRepository,
+  @Autowired val auditService: AuditService,
 ) {
 
   @GetMapping("/getMockOrderSummary/{orderId}")
   fun getMockOrderSummary(
+    authentication: Authentication,
```

Comment:
In main we've adjusted it so that the Authentication header is required. This sensibly overwrites what we've got in main; we might have to adjust the tests to consume this.

Just to check - I think the API app constructs this from the Auth header it receives?